code/game/Game.Debug.cs

```diff
@@ -166,4 +167,41 @@ public static void SpawnNpcCMD(string npcName)
 
         npc.Position = tr.EndPosition;
     }
+
+    [ConCmd.Server("cinema.player.job.set")]
+    public static void SetJob(string jobName, string playerName = "")
+    {
+        if (!ValidateUser(ConsoleSystem.Caller.SteamId)) return;
+
+        if (ConsoleSystem.Caller.Pawn is not Player player) return;
+
+        var target = player;
+
+        if (!string.IsNullOrEmpty(playerName))
+        {
+            var clients = Game.Clients.Where(x => x.Name.ToLower().Contains(playerName));
+
+            if (clients.Count() is <= 0 or > 1) return;
+
+            var client = clients.FirstOrDefault();
+
+            if (client == ConsoleSystem.Caller.Client || client.Pawn is not Player given) return;
+
+            target = given;
+        }
+
+        var details = Jobs.JobDetails.All.FirstOrDefault(x => x.Name.ToLower().Contains(jobName.ToLower()));
+
+        if (details == null) return;
+
+        target.SetJob(details);
+    }
+
+    [ConCmd.Client("cinema.player.job.debug.client")]
+    public static void DebugJobClient()
+    {
+        if (ConsoleSystem.Caller.Pawn is not Player player) return;
+        Log.Info($"Job: {player.Job.JobDetails.Name}");
+        Log.Info($"Abilities: {player.Job.JobDetails.Abilities.ToNiceBinary()}");
```

Comment:
If JobDetails.Abilities were of type JobAbilities as suggested in my comment on Jobs.cs, there would be an easier way to enumerate job abilities here, making the ToNiceBinary extension obsolete. You'd also get access to human-readable names.